src/app-directory/specification/appd.yaml

```
@@ -607,7 +607,8 @@ components:
       properties:
         displayName:
           type: string
-          description: An optional display name for the intent that may be used in UI instead of the name.
+          description: Optional display name for the intent. Deprecated in favour of the Intent name, which is common amongst all apps that support it, where the display name may vary.
```

Should these usages of `intent` have the same capitalization? Is the capitalized one referencing the Intent concept? Sorry I'm still kind of fuzzy on when capitalization is appropriate for some of these words.

---

I think it should be lowercase...

---

Lets go with lowercase - its that way in the glossary and I think appropriate in all cases other than when we are specifically discussing the concept of Intents.

 ```suggestion
          description: Optional display name for the intent. Deprecated in favour of the intent name, which is common amongst all apps that support it, where the display name may vary as it is defined in the app's AppD record.
```